scripts/src/buildandtest/buildandtest.py

```diff
@@ -103,24 +103,30 @@ def main():
                         help="Name of the chart verifier image")
     parser.add_argument("-s", "--sha-value", dest="sha_value", type=str, required=True,
                         help="Image sha value to test")
-    parser.add_argument("-v", "--verifier-version", dest="verifier_version", type=str, required=True,
+    parser.add_argument("-v", "--verifier-version", dest="verifier_version", type=str, required=False,
                         help="New version of chart verifier")
+    parser.add_argument("-b","--build-only",dest="build_only",type=str, required=False,
+                        help="set to any value to build only and not test")
 
 
     args = parser.parse_args()
-
-    image_id = args.image_name + ":" + args.sha_value[:7]
+    image_tag = args.sha_value[:7]
```

Comment:
I would suggest use a variable for '7' and give a proper name so later contributers do not need to guess the purpose of this value.

Reply:
Good spot, I couldn't remember what the 7 was for so had to do some testing :-). It was to change the sha from long form to short form so after a bit of investigation I found a better way to do it. In build.yaml:  

```
commit_sha=$(git rev-parse --short HEAD)
ve1/bin/build-and-test --image-name=${{steps.check_version_in_PR.outputs.PR_release_image}} --sha-value=$commit_sha --verifier-version=${{steps.check_version_in_PR.outputs.PR_version}}
```

this way the 7 is no longer needed